client/src/Components/mainServices/MainServicesCard.tsx

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable max-len */
+import React from 'react';
+import { Card } from 'antd';
+import 'antd/dist/antd.css';
+import './mainServices.css';
+
+interface serviceProps {
+  service: string,
+  imgLink: number,
+}
+
+function MainServicesCard({ service, imgLink }: serviceProps) {
+  return (
+    <Card.Grid className="main-services-card">
+      <img src={`../../../images/${imgLink + 1}.png`} alt="main service we provide" className="main-services-crd-image" />
```

Comment:
```suggestion
      <img src={`/images/${imgLink + 1}.png`} alt="main service we provide" className="main-services-crd-image" />
```
anything inside public simply have a path of /etc
and we can change alt to service text